extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -128,6 +129,42 @@ export const LIFECYCLE_STAGES = {
 };
 
 
+/**
+ * Utility function that ensures any error thrown is handled by optional
+ * onError handler (if none provided or handler throws, error is swallowed).
```

Comment:
For an abundance of clarity, can this be "if none provided or handler throws, error is swallowed and undefined is returned"?